Makefile

```diff
@@ -116,8 +117,17 @@ endif
 	go generate ./pkg/... ./cmd/...
 	./scripts/sync-up-helm-chart.sh
 
-push:
-	$(DOCKER) push $(REGISTRY)/$(TARGET):$(GIT_VERSION)
+push: container
+
+	if [ "$(TRAVIS_PULL_REQUEST)" = "false" ]; \
+	then \
+		$(DOCKER) login -u "$(QUAY_USERNAME)" -p "$(QUAY_PASSWORD)" quay.io; \
+	fi
+
+        ifeq ($(TRAVIS_PULL_REQUEST),)
```

Comment:
How come you've chosen a make conditional here but use shell conditionals elsewhere?

Reply:
Updated to use shell.